### ISSUE TITLE
Orma-Processor error when Not contains @Column nor @PrimaryKey in @Table-defined subclass

### ISSUE THREAD

**Issue by keima** [gfx/Android-Orma#220]
When super-class and sub-class are defined below, Orma-Processor outputs error "[OrmaProcessor] No @Column nor @PrimaryKey is defined".

Perchance, sub-class must have @Column ?

----
```Child.java
@Table
public class Child extends Parent {
  // nothing
}
```

```Parent.java
public class Parent {
  @Column(value = BaseColumns._ID)
  @PrimaryKey(autoincrement = true)
  public long id;

  @Column
  public String title;
}
```

**Comment by gfx** [comment]
Thanks to reporting the bug :smile: 

Fixed this issue and released v2.3.3 :dancers:

**Comment by keima** [comment]
Excellent! :+1:
